### DISCUSSION TITLE
Catalan Translation

### DISCUSSION THREAD

**Discussion by amagraphics** [ChristopherKlay/StadiaEnhanced#60] [Translations]
case 'ca': // https://github.com/ChristopherKlay/StadiaEnhanced/discussions/60
        translate_load = {
            default: 'Per defecte',
            native: 'Nativa',
            hide: 'Amaga',
            show: 'Mostra',
            total: 'Total',
            visible: 'Visible',
            hidden: 'Amagat',
            enabled: 'Activat',
            disabled: 'Desactivat',
            auto: 'Automàtic',
            manual: 'Manual',
            all: 'Tots',
            locked: 'Blocats',
            complete: 'Complet',
            incomplete: 'Incomplet',
            games: 'Jocs',
            allgames: 'Tots els jocs',
            leavepro: 'Aviat deixaran Stadia Pro',
            bundles: 'Paquets',
            addons: 'Complements',
            wishlist: 'Llista de desitjos',
            responsive: 'Responsiu',
            windowed: 'Mode de finestra',
            fullscreen: 'Pantalla completa',
            onsale: 'En venda',
            prodeals: 'Ofertes de Stadia Pro',
            userprofile: 'El meu perfil',
            usermedia: 'Captures',
            searchbtnbase: 'Cerca a',
            avatarpopup: 'URL de l\'avatar nou (buit per defecte):',
            date: 'Data',
            time: 'Temps',
            sessiontime: 'Temps de sessió',
            codec: 'Còdec',
            resolution: 'Resolució',
            hardware: 'Maquinari',
            software: 'Programari',
            trafficsession: 'Trànsit de sessió',
            trafficcurrent: 'Trànsit actual',
            trafficaverage: 'Trànsit mitjà',
            packetloss: 'Paquets perduts',
            framedrop: 'Fotogrames perduts',
            latency: 'Latència',
            jitter: 'Jitter Buffer',
            decodetime: 'Temps de descodificació',
            compression: 'Compressió',
            network: 'Xarxa',
            bitrate: 'Taxa de bits',
            streammon: 'Monitor de retransmissió',
            stream: 'Retransmissió',
            session: 'Sessió',
            extdetail: 'Més detalls',
            maxresolution: 'Resolució màxima',
            fps: 'Fotogrames per segon',
            testdiscl: '<b>Avís:</b> Aquest joc encara no s'ha provat.',
            datadiscl: 'Aquest és el màxim de fotogrames per segon aconseguit quan es juga un joc en mode 4K (cal ser subscriptor Pro).\
                En els jocs amb l'opció de seleccionar entre resolució / velocitat de fotogrames, es va triar la resolució. \
                Aquestes dades les proporciona <a href="https://twitter.com/OriginaIPenguin" target="_blank">@OriginaIPenguin</a> \
                i es pot trobar la base de dades completa <a href="https://airtable.com/shr32bmiOThVvSGar/tblAeJTnP2bzZyews" target="_blank">aquí</a>.'
            noteOne: 'Mode 4K',
            noteTwo: 'Selecciona entre 30 o 60 FPS',
            noteThree: '60 FPS en el mode 1080p',
            noteFour: '30 FPS en el mode 1080p',
            noteFive: 'No és compatible amb el mode 4K',
            unsupported: 'No admés',
            crossfriends: 'No permet el joc creuat entre plataformes',
            community: 'Comunitat',
            speedtest: 'Prova de velocitat',
            quickaccess: 'Accés ràpid',
            messages: 'Missatges',
            comfeature: 'Funcions de la comunitat',
            avatar: 'Avatar',
            interface: 'Interfície',
            shortcut: 'StadiaIcons',
            shortcuttitle: 'Instal·la una drecera per a',
            shortcutdesc: 'Permet instal·lar una drecera per a un joc al dispositiu',
            stadiastats: 'StadiaStats',
            stadiastatsopen: 'Veure a StadiaStats.GG',
            stadiastatsdesc: 'Permet les dreceres directes a les estadístiques dels jocs, l\'enllaç al vostre perfil i el sistema de cerca d\'altres jugadors/es a stadiastats.gg.',
            stadiadatabase: 'Stadia Database',
            stadiadatabasedesc: 'Mostra una secció de "Més detalls" a la pàgina de la botiga dels jocs, que mostra la freqüència de fotogrames, la resolució i molt més.',
            gridsize: 'Tamany de la quadrícula',
            griddesc: 'Canvia la quantitat de jocs per fila a la biblioteca.',
            clock: 'Rellotge',
            clockdesc: 'Mostra l\'hora actual a la llista d\'amics, com a superposició del joc, o ambdues coses.',
            friendslist: 'Llista d\'amics',
            igoverlay: 'Superposició dins del joc',
            listoverlay: 'Llista i superposició',
            filter: 'Filtre de jocs',
            filterdesc: 'Permet ordenar la biblioteca amagant jocs. El filtre es pot alternar mitjançant el símbol, a la part superior dreta, a sobre dels jocs de la biblioteca.',
            invitebase: 'Copia l\'enllaç d\'invitació',
            inviteactive: 'Copiat!',
            gamelabel: 'Etiquetes de joc',
            gamelabeldesc: 'Elimina etiquetes com "Pro" dels jocs del menú principal.',
            dimoverlay: 'Superposició atenuada',
            dimoverlaydesc: 'Elimina l'efecte d'atenuació en obrir el menú de Stadia durant el joc.',
            homegallery: 'Galeria d\'usuari',
            homegallerydesc: 'Amaga l\'àrea "Captures i estats del joc" de la part inferior de la pantalla d\'inici.',
            quickprev: 'Vista prèvia del missatge',
            quickprevdesc: 'Amaga la previsualització del missatge a la llista d\'amics.',
            quickrep: 'Resposta ràpida',
            quickrepdesc: 'Amaga l\'opció de resposta ràpida als xats.',
            offlinefriend: 'Amics fora de línia',
            offlinefrienddesc: 'Amaga els amics fora de línia a la llista d\'amics.',
            invisiblefriend: 'Amics invisibles',
            invisiblefrienddesc: 'Amaga els amics amb estat en línia desconegut a la llista d\'amics.',
            notification: 'Notificacions',
            notificationdesc: 'Mostra una notificació quan Stadia Enhanced s'actualitza a una nova versió ("Auto" s'amaga al cap de 5 segons, "Manual" es manté fins a la interacció de l'usuari).',
            streammode: 'Mode de d\'emissió en directe',
            streammodedesc: 'Permet fer que alguns elements (com ara la llista d\'amics) no es puguin llegir durant l\'emissió en directe (mitjançant eines com OBS / Discord).',
            catprev: 'Vista prèvia de la categoria',
            catprevdesc: 'Amaga les etiquetes de categoria quan es passa el cursor per sobre d\'un joc.',
            streammondesc: 'Activa-ho per iniciar el monitor sempre que comenci un joc.',
            resolutiondesc: 'La resolució específica per als jocs. 1440p i 2160p requereixen VP9.',
            codecdesc: 'El còdec utilitzat per als jocs.',
            confirmreset: 'Segur que vols restablir la configuració?',
            statistics: 'Estadístiques',
            gamesfinished: 'Jocs acabats',
            achievementsunlocked: 'Assoliments desbloquejats',
            totalPlayTime: 'Temps de joc total',
            splitstore: 'Divideix les llistes de la botiga',
            splitstoredesc: 'Divideix les llistes de la botiga en dues columnes per obtenir una millor visió.',
            inlineimage: 'Vista prèvia de la imatge',
            inlinedesc: 'Substitueix els enllaços d\'imatge per a formats de fitxer habituals (jpg/gif/png) amb una vista prèvia.',
            familyelements: 'Opcions de compartició familiar',
            familyelementsdesc: 'Amaga les opcions "Comparteix aquest joc amb la família."',
            donations: 'Donacions',
            reportbug: 'Reporta un error',
            exportset: 'Exporta la configuració',
            importset: 'Importa la configuració',
            importerror: 'El fitxer que intenteu obrir no conté un perfil vàlid de Stadia Enhanced.',
            resetsettings: 'Restableix la configuració'
        }

**Comment by ChristopherKlay** [comment]
There was a single new line added to the translations page after the patch;

> "responsive": "Responsive"

Once that's added, i will include it for `1.8.1`. Thank you so much for it!

**Reply by amagraphics** [reply]
Edited! And thank you very much for this amazing extension :)

**Comment by ChristopherKlay** [comment]
One last needed change, due to [this request](https://www.reddit.com/r/Stadia/comments/kosrac/stadia_enhanced_changelog_180_new_year_new_update/ght3tco/?utm_source=reddit&utm_medium=web2x&context=3) of adding a menu element for shortcuts;
```
"shortcut": "Shortcuts",
"shortcuttitle": "Install a shortcut for",
"shortcutdesc": "Allows you to install a shortcut for a game on your device",
```

**Reply by amagraphics** [reply]
Added!

**Comment by ChristopherKlay** [comment]
Small update of the translation file for `1.8.4` [here](https://github.com/ChristopherKlay/StadiaEnhanced/blob/master/translations.md#new-content).
No stress however; i'll update translation when available (:

**Reply by amagraphics** [reply]
Edited!

I'll leave ya the new strings here too:
"games": "Jocs",
"bundles": "Paquets",
"addons": "Complements",
"hardware": "Maquinari",
"software": "Programari",
"decodetime": "Temps de descodificació",

**Comment by amagraphics** [comment]
@ChristopherKlay Greetings! The catalan translation appear to be broken now :S 
![Screenshot_1](https://user-images.githubusercontent.com/42438325/104723393-7d2b4800-572f-11eb-823a-2e62448caa97.jpg)

**Reply by ChristopherKlay** [reply]
Just checked; 

> enhanced.js:3035 Stadia Enhanced Error loading translation file: SyntaxError: Unexpected string in JSON at position 543

For some reason there's two `,` missing in the translation file that are present in the previous version. I'm honestly not sure how that happend, but it will be fixed in `1.8.6`. I'm also currently working on a quick debugging function that checks all translations, so these issues don't happen again in the future.

Thanks for letting me know (:

**Reply by ChristopherKlay** [reply]
Update: `1.8.6` just went live on GitHub and should update on the chrome store soon as well, fixing this issue (:

**Comment by amagraphics** [comment]
"wishlist": "Llista de desitjos",
"userprofile": "El meu perfil",
"inlineimage": "Vista prèvia de la imatge",
"inlinedesc": "Substitueix els enllaços d'imatge per a formats de fitxer habituals (jpg/gif/png) amb una vista prèvia.",
"familyelements": "Opcions de compartició familiar",
"familyelementsdesc": "Amaga les opcions "Comparteix aquest joc amb la família.",

**Reply by amagraphics** [reply]
@ChristopherKlay here are the new strings, thanks for your hard work! :)

**Reply by ChristopherKlay** [reply]
Thank you for the quick translations!

Will push out a hotfix later today and include these (:

**Comment by amagraphics** [comment]
@ChristopherKlay updated OP strings, here are the new ones:

```
"total": "Total",
"gamelabel": "Etiquetes de joc",
"gamelabeldesc": "Elimina etiquetes com 'Pro' dels jocs del menú principal.",
```

**Comment by amagraphics** [comment]
@ChristopherKlay Thank you for the new update. A lot of amazing additions, as usual! I've updated the new strings on the OP, if you need to, I can adapt them to the new template tomorrow :)

**Reply by ChristopherKlay** [reply]
Will be included in `1.11.1` later today and i updated the post to the new version including changes (:

**Comment by amagraphics** [comment]
@ChristopherKlay updated strings ;)

**Reply by ChristopherKlay** [reply]
Will be included in the next patch (:
